IFC4x3/Properties/e/ExcavationClassification_0M$dI8P5T3I8$lTN_LlVyH/DocProperty.xml

```diff
@@ -0,0 +1,2 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Should this not be modelled with [Classification Association](https://bsi-infraroom.github.io/IFC-Documentation-Tunnel/4_4_0_0/general/HTML/link/classification-association.htm)?

Reply:
Rename to ExcavationClass. @JontyDunn or @hessf-ilf might improve the description, e.g. with an example. Right now it is "Specifies an excavation class used in underground excavation."